package.json

```diff
@@ -18,6 +18,7 @@
   "scripts": {
     "bootstrap": "lerna bootstrap",
     "format": "lerna run format",
+    "format-text": "prettier --write --prose-wrap always --print-width 80 \"./*.md\" \"./docs/**/*.md\" && lerna run format-text",
```

Comment:
This recurses, no?

Reply:
No:

```
lerna run <script>

Run an npm script in each package that contains that script
```

i.e. `lerna run format-text` runs `yarn format-text` for each folder in `packages/` only.

Reply:
Ah, right. `lerna run` not `yarn run`.